scripts/scr_income/scr_income.gml

```diff
@@ -61,8 +61,8 @@ function scr_income() {
 
 	if (obj_ini.fleet_type=1){
 	    with(obj_star){
-	        if (string_count("Monastery",p_feature[1])=1){obj_controller.income+=10;instance_create(x,y,obj_temp1);}
-	        if (string_count("Monastery",p_feature[2])=1){obj_controller.income+=10;instance_create(x,y,obj_temp1);}
+	        if (planet_feature_bool(p_feature[1], P_features.Monastery)==1){obj_controller.income+=10;instance_create(x,y,obj_temp1);}
+	        if (planet_feature_bool(p_feature[2], P_features.Monastery)==1){obj_controller.income+=10;instance_create(x,y,obj_temp1);}
```

Comment:
why do we check for monastery both at index 1 and 2? can the planet have 2 monasteries?

Reply:
p_feature[1] and p_feature[2] refer to the array contaning the features for planet1 and planet2 so this is checking both planets in the system

Reply:
hmm, but what if there are 4 planets? or if the monastery is on planet num3?

Reply:
@GalacticChimp base player star system always has two planets but now thinking about it will edit for future versions where chapters without monastery can build a new one.